cabby/geo/walk.py

```diff
@@ -107,15 +120,16 @@ def get_start_poi(map: map_structure.Map, end_point: GeoDataFrame) -> \
     outer_circle_graph = ox.truncate.truncate_graph_dist(
         map.nx_graph, dest, max_dist=2000, weight='length')
 
-    # Get graph that is too close (less than 200 meter path distance)
+    # Get graph that is too close (less than 400 meter path distance)
     inner_circle_graph = ox.truncate.truncate_graph_dist(
-        map.nx_graph, dest, max_dist=200, weight='length')
+        map.nx_graph, dest, max_dist=400, weight='length')
 
     outer_circle_graph_osmid = list(outer_circle_graph.nodes.keys())
     inner_circle_graph_osmid = list(inner_circle_graph.nodes.keys())
 
     osmid_in_range = [
-        osmid for osmid in outer_circle_graph_osmid if osmid not in inner_circle_graph_osmid]
+        osmid for osmid in outer_circle_graph_osmid if osmid not in 
+				inner_circle_graph_osmid]
```

Comment:
Reformat multiline comprehension according to style guide: https://google.github.io/styleguide/pyguide.html#274-decision

(Break at the 'if'.)